docs/Symbols.md

```diff
@@ -10,4 +10,125 @@ The following are also listed:
 ![An XML document that references two grammars. The referenced grammars are listed in the outline](./images/Symbols/ShowReferencedGrammars.png)
 
 This option has no effect if symbols are disabled through `xml.symbols.enabled`.
-The displayed symbols are affected by `xml.symbols.maxItemsComputed`
\ No newline at end of file
+The displayed symbols are affected by `xml.symbols.maxItemsComputed`
+
+## xml.symbols.filters
+
+By default Outline (symbols) display `DOM elements`, `processing instruction` and `DTD element, entity, attribute list declarations`.
+
+The DOM attributes and text nodes are not displayed in the outline. Indeed to ensure good performance, symbols must be limited; displaying attributes and text nodes in the outline could generate a lot of symbols.
+
+Here a sample Outline with maven `pom.xml`:
+
+![Outline of pom.xml without filter](images/Symbols/SymbolsPOMWithoutFilter.png)
+
+In maven context, the text nodes information are important and we should prefer showing text nodes in the Outline:
+
+![Outline of pom.xml with text filter](images/Symbols/SymbolsPOMWithTextFilter.png)
+
+In Spring context, the @Id attribute information are important and we should prefer showing those attributes in the Outline:
+
+![Outline of Spring beans with @id filter](images/Symbols/SymbolsBeansWithIdAttrFilter.png)
+
+In other words, displaying attributes or text nodes depends on the XML file.
+
+`xml.symbols.filters` gives the capability to define filter (to include or exclude) some attributes, some text nodes for a given XML file kind.
+
+Symbols filter are composed with:
+
+  * `pattern` (required) : a regular expression matching the file names to which this filter should apply.
+  * `expressions` (required) : defines a list of expression. An expression is composed by:
+    **  `xpath` (required) : defines a basic xpath to declare the attribute, the text node which is concerned by the expression.
```

Comment:
In order to get the second level of indent:
```md
 * Level 1 of indent
    * Level 2 of indent
```

Reply:
fixed